scripts/validate.py

```diff
@@ -51,6 +52,11 @@ def run(global_config):
             "Expecting non-empty IGNORED_FILES. Please check if ignored_files_manifest.csv is configured correctly!!!"
         )
 
+    logger.info(
```

Comment:
lol another one. Could you add this log under an `if FORCE_CREATE_MANIFEST` . So if true then let user know with the logs